### PR TITLE
Rename fbx to model serializers

### PR DESCRIPTION
**Pull request by JulianGro**
This PR moves all model serializers from the `fbx` folder to the `model-serializers` folder.
I guess High Fidelity was just always too lazy to clean this up. It seem somewhat important to do this cleanup though, as a new developer would hardly be searching for the gltf or obj serializers in the "fbx" folder.

Builds and runs at least on my Linux Mint 19.3 workstation.

### CODE REVIEW

File: libraries/model-serializers/src/GLTFSerializer.cpp [hunk 1]
```diff
@@ -1863,8 +1863,8 @@ void GLTFSerializer::setHFMMaterial(HFMMaterial& hfmMat, const GLTFMaterial& mat
 
 template<typename T, typename L>
 bool GLTFSerializer::readArray(const hifi::ByteArray& bin, int byteOffset, int count,
-                           QVector<L>& outarray, int accessorType, bool normalized) {
-    
+                           QVector<L>& outarray, int accessorType) {
```
Comment (HifiExperiments) [inline libraries/model-serializers/src/GLTFSerializer.cpp:1866]: this looks like we’ve lost some stuff here

File: libraries/model-serializers/src/GLTFSerializer.h [hunk 2]
```diff
@@ -519,15 +519,15 @@ struct GLTFAccessor {
...
+
         }
     };
     int bufferView;
     int byteOffset { 0 };
     int componentType; //required
     int count; //required
     int type; //required
-    bool normalized { false };
+    bool normalized{ false };
```
Comment (HifiExperiments) [inline libraries/model-serializers/src/GLTFSerializer.h:530]: put back space

**Review by HifiExperiments — CHANGES_REQUESTED** [review]
I think it’s a good idea to have these changes in their own PR, separate from the general serializer overhaul.  just a few comments

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-8568400-_8573425f.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-8568400-.tar_8573425f.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-8568400-.tar_8573425f.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-8568400-_8573425f.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-8568400-_8573425f.exe

**Comment by ctrlaltdavid** [comment]
Android build failed with a cmake error: https://github.com/vircadia/vircadia/runs/2355646699

**Comment by JulianGro** [comment]
Sorry about the mess. Half of that was a GitAhead bug and the other half was me somehow ending up with an outdated GLTFSerializer.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a-.tar_f6d9cd6f.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a-_f6d9cd6f.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a-.tar_f6d9cd6f.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a-_f6d9cd6f.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a-_f6d9cd6f.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-PR1178-78e791a_f6d9cd6f.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1178/Vircadia-Quest-Alpha-PR1178-78e791a_f6d9cd6f.apk

**Review by HifiExperiments — APPROVED** [review]

**Comment by digisomni** [comment]
A good test would be to just run this on Windows and see if everything works as usual.

**Review by digisomni — APPROVED** [review]
Works fine for me on Windows.

**Pull request merged**